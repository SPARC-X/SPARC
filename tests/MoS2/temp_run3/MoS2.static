Total free energy (Ha): -8.908400729930021E+01
Atomic forces (Ha/Bohr):
 -2.3517695819E-02   3.8929182444E-02   1.0727287402E-01
  1.2527297671E-02  -1.1081915467E-02  -6.8716132917E-02
  1.0990398148E-02  -2.7847266977E-02  -3.8556741106E-02
Stress (Ha/Bohr**2): 
  9.6396289058E-05  -5.1510735123E-04 
 -5.1510735123E-04   4.6313314574E-03 
Stress equiv. to all periodic (GPa): 
  9.4535891115E-02  -5.0516604887E-01 
 -5.0516604887E-01   4.5419491834E+00 
