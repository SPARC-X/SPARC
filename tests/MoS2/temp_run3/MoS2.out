***************************************************************************
*                       SPARC (version Apr 23, 2021)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Mon May 10 20:59:57 2021                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 6.1 30 6.1 
LATVEC:
-0.5 0.8666 0 
0 0 1 
1 0 0 
FD_GRID: 51 250 51
FD_ORDER: 12
BC: P D P
KPOINT_GRID: 3 1 3
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.0009999935878
CHEB_DEGREE: 49
NSTATES: 20
EXCHANGE_CORRELATION: GGA_PBE
CALC_STRESS: 1
CHEFSI_BOUND_FLAG: 0
MAXIT_SCF: 100
MINIT_SCF: 3
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-08
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 0
PRINT_EIGEN: 0
PRINT_DENSITY: 0
OUTPUT_FILE: MoS2
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 5
NP_BAND_PARAL: 1
NP_DOMAIN_PARAL: 1 9 1
NP_DOMAIN_PHI_PARAL: 2 12 2
EIG_SERIAL_MAXNS: 2000
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  48
Mesh spacing in x-direction        :  0.119608 (Bohr)
Mesh spacing in y-direction        :  0.12 (Bohr)
Mesh spacing in z direction        :  0.119608 (Bohr)
Number of symmetry adapted k-points:  5
Output printed to                  :  MoS2.out
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  26
Atom type 1  (valence electrons)   :  Mo 14
Pseudopotential                    :  Mo.psp8
Pseudocharge radii of atom type 1  :  6.70 6.72 6.70 (x, y, z dir)
Number of atoms of type 1          :  1
Atom type 2  (valence electrons)   :  S 6
Pseudopotential                    :  S.psp8
Pseudocharge radii of atom type 2  :  6.70 6.72 6.70 (x, y, z dir)
Number of atoms of type 2          :  2
Estimated total memory usage       :  5.97 GB
Estimated memory per processor     :  127.43 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -2.9569664513E+01        1.436E-01        62.803
2            -2.9652955512E+01        2.967E-01        16.687
3            -2.9689568116E+01        7.684E-02        16.488
4            -2.9693136212E+01        5.839E-02        16.420
5            -2.9692174162E+01        7.713E-02        16.621
6            -2.9694615136E+01        1.348E-02        16.571
7            -2.9694637160E+01        8.573E-03        16.345
8            -2.9694665994E+01        3.340E-03        16.228
9            -2.9694668632E+01        9.454E-04        15.950
10           -2.9694668533E+01        1.445E-03        15.732
11           -2.9694669071E+01        4.198E-04        15.744
12           -2.9694669117E+01        3.277E-05        15.610
13           -2.9694669231E+01        4.400E-05        15.392
14           -2.9694669204E+01        1.976E-05        15.345
15           -2.9694669142E+01        8.204E-06        15.225
16           -2.9694669143E+01        3.570E-06        15.182
17           -2.9694669152E+01        1.615E-06        15.115
18           -2.9694669073E+01        1.421E-06        14.963
19           -2.9694669120E+01        1.180E-06        14.938
20           -2.9694669100E+01        5.377E-07        15.061
Total number of SCF: 20    

k = [-0.333, 0.000, -0.333]
Occupation of state 18 = 0.000000000000000.
Occupation of state 20 = 0.000000000000000.
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -2.9694669100E+01 (Ha/atom)
Total free energy                  : -8.9084007299E+01 (Ha)
Band structure energy              : -2.1030683015E+01 (Ha)
Exchange correlation energy        : -1.2270360045E+01 (Ha)
Self and correction energy         : -2.1198465860E+02 (Ha)
Entropy*kb*T                       : -3.3304221091E-07 (Ha)
Fermi level                        : -1.7626958878E-01 (Ha)
RMS force                          :  7.8684445776E-02 (Ha/Bohr)
Maximum force                      :  1.1651623391E-01 (Ha/Bohr)
Time for force calculation         :  0.206 (sec)
Maximum stress                     :  4.6313314574E-03 (a.u.)
Time for stress calculation        :  0.457 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  365.420 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Collaborators: J.E. Pask (LLNL), A.J. Medford (GT), E. Chow (GT)    *
*  Acknowledgements: U.S. DOE (DE-SC0019410); U.S. NSF (1333500,1553212)  *
***************************************************************************
                                                                           
